lib/tasks/elasticsearch.rake

```diff
@@ -1,32 +1,18 @@
-require "elasticsearch/rails/tasks/import"
-
 namespace :elasticsearch do
-  task :drop do
-    Rubygem.__elasticsearch__.client.indices.delete index: Rubygem.index_name
+  task drop: :environment do
+    Rubygem.searchkick_index.delete
   end
-  task :create do
-    Rubygem.__elasticsearch__.create_index! force: true
-    Rubygem.__elasticsearch__.refresh_index!
+  task create: :environment do
+    Rubygem.searchkick_index.create
   end
 
-  task :import_alias do
```

Comment:
Online async import is supported in Searchkick by default https://github.com/ankane/searchkick#parallel-reindexing, but I wasn't able to setup it locally without adding Redis dependency. Anyway since we don't need reindexing currently, I just made it simple.

In production we can call:

```ruby
index = Rubygem.reindex(mode: :async)
# wait once done
Rubygem.searchkick_index.promote(index[:index_name])
```

Reply:
As I mentioned, we can do the same and it is built in. Searchkick utilizes ActiveJob we doesn't really use currently. But I have plan to move all `.delay` calls to proper background jobs soon as well. For now we can do async reindexing as mentioned. Not reindexing is needed when deployed since the using index is exactly the same (checked manually via API).

Reply:
> For now we can do async reindexing as mentioned.

Does this work without redis? I think somewhere you commented that async needs redis.

Reply:
Yes, redis is needed just for optional progress tracking AFAIK.